### PR TITLE
bug/589_no_ckan_resources_in_organization_show

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #589 
* 100% unit tests passed:
```
Tests run: 59, Failures: 0, Errors: 0, Skipped: 0
```
* (unofficial) e2e tests passed:
```
time=2015-11-02T14:41:35.027CET | lvl=INFO | trans=1446471690-765-0000000000 | function=getEvents | comp=Cygnus | msg=com.telefonica.iot.cygnus.handlers.OrionRestHandler[150] : Starting transaction (1446471690-765-0000000000)
time=2015-11-02T14:41:35.030CET | lvl=INFO | trans=1446471690-765-0000000000 | function=getEvents | comp=Cygnus | msg=com.telefonica.iot.cygnus.handlers.OrionRestHandler[232] : Received data ({  "subscriptionId" : "51c0ac9ed714fb3b37d7d5a8",  "originator" : "localhost",  "contextResponses" : [    {      "contextElement" : {        "attributes" : [          {            "name" : "temperature",            "type" : "centigrade",            "value" : "26.5"          }        ],        "type" : "Room",        "isPattern" : "false",        "id" : "Room1"      },      "statusCode" : {        "code" : "200",        "reasonPhrase" : "OK"      }    }  ]})
time=2015-11-02T14:41:35.032CET | lvl=INFO | trans=1446471690-765-0000000000 | function=getEvents | comp=Cygnus | msg=com.telefonica.iot.cygnus.handlers.OrionRestHandler[255] : Event put in the channel (id=84514916, ttl=10)
time=2015-11-02T14:41:36.589CET | lvl=INFO | trans=1446471690-765-0000000000 | function=persistOne | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionCKANSink[207] : [ckan-sink] Persisting data at OrionCKANSink (orgName=def_serv_deleteme_1, pkgName=def_serv_deleteme_1_def_serv_path_delete_2, resName=room1_room, data=1446471695032,2015-11-02T13:41:35.32Z,Room1,Room,temperature,centigrade,"26.5",[])
time=2015-11-02T14:41:41.629CET | lvl=INFO | trans=1446471690-765-0000000000 | function=process | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionSink[178] : Finishing transaction (1446471690-765-0000000000)
..
..
```
ckan_second_dataset
* Assignee @fgalan, but LGTM from @acs is also required

### CODE REVIEW

**Comment by acs** [comment]
LGTM, I need to do the same change in the ChanChan App to support CKAN 2.5 API:

Client: https://github.com/Bitergia/fiware-chanchan/commit/4d91f4f6e39a48762d53ac2dba3e36c0fb25e647#diff-7b6578e95977e4e70e6e4e9de094b0c9L90

Server: https://github.com/Bitergia/fiware-chanchan/commit/c9d8422b3ea0ec834181c1a4c7788210b8a4bc8b#diff-012b854aa34679ef34863bc62d1edf9bR9

File: src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java [hunk 1]
```diff
@@ -355,8 +355,10 @@ private void populatePackagesMap(JSONArray packages, String orgName) throws Exce
             LOGGER.debug("Package found in CKAN, now cached (orgName=" + orgName + " -> pkgName/pkgId=" + pkgName
                     + "/" + pkgId + ")");
             
-            // get the resources
-            JSONArray resources = (JSONArray) pkg.get("resources");
+//            // get the resources
+//            JSONArray resources = (JSONArray) pkg.get("resources");
```
Comment (fgalan) [inline src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java]: Leftover?

Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/backends/ckan/CKANCache.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/e6a6b43f5d2506c4b49669e7f108c6a9e2ae66cc

**Comment by fgalan** [comment]
LGTM

**Pull request merged**